cmd/in/main.go

```diff
@@ -170,14 +170,8 @@ func rootfsFormat(dest string, req InRequest, image v1.Image) {
 	}
 
 	env := cfg.Config.Env
-	if len(env) == 0 {
```

Comment:
What was the cue for removing these? :thinking: 

It looks like the tests pass either way, but they also passed before, so I wonder whether this was important and if we need test coverage for it or if I was just kind of guessing that it'd be useful.

Reply:
`config.ContainerConfig` was removed in [this commit](https://github.com/google/go-containerregistry/commit/2be26e624e32c4b3de62287e153e5e382a79b7c5), and moved to `/pkg/legacy/config. LayerConfigFile`. I didn't find a reference to `LayerConfigFile` here, so I removed the reference to `ContainerConfig` altogether.

Reply:
Good enough for me. If someone complains we can just introduce legacy compatibility and add a test. :+1: